chapters/Chapter 2 — Dressara.md

```diff
@@ -0,0 +1,94 @@
+
+She hadn’t been hard to track down. She’d only changed Face once. Faces records could only be unsealed with a half-dozen odd warrants, but Leanne didn’t need the records: while Leanne didn’t think she was all that great at tracking people down through their identity changes, she had to acknowledge that she was far better at it than any of her peers. She often hardly needed to research; she’d simply guess what people would do with their new identities, and frequently, she’d be right. 
+
+Leanne had guessed a Dressaran woman like Delphine wouldn’t miss out on the lavish parties her relations threw, even if she would now have to attend as a guest in order not to be associated with her former self and her former scandals. She had been right.
+
+“Ma’am,” said Leanne, eyes downcast, as she handed Delphine her glass of wine. Delphine was on the outskirts of the party, observing from the distance. Leanne guessed she was wistful about a past she could never fully rejoin. Then again, if Dressara were her past, Leanne wouldn’t want to rejoin it.
+
+Delphine took the glass, her fingers winding around it delicately, her deep red fingernails clicking gently against it.
+
+She looked at Leanne, glanced her up and down. The corner of her lips turned up a bit further.
+
+“You look stunning, obviously,” said Leanne.
+
+A soft chuckle escaped Delphine. “Obviously,” she agreed. “Not that you’ve done poorly with yourself.”
+
+“I’m not different than the others,” said Leanne, nodding her head towards the other servants, who were attending a loud crowd of walking primping Dressaran men.
+
+Delphine glanced at the others, and back at Leanne. “I don’t think so.” Her smirk faded a bit as she glanced at the blood red of her fingernails. “Not that theres’s anything wrong with them, they’re just…”
+
+Her eyes hardened and turned to Leanne. “Not you.”
+
+Leanne’s breath wanted to catch. “Not me, no.”
+
+“Then who are you?” asked Delphine, closing the distance between them. “Why come here as a servant? They wouldn’t know the difference if you came as Queen of Dressara.” 
+
+Leanne thought that might be a bit of a stretch. Dressara had never had a queen—neither had any planet in Q729.
+
+“You knew the difference,” said Leanne.
+
+“I’m not from Dressara,” said Delphine.
+
+“Yes, you are,” said Leanne.
+
+Delphine’s smirk returned. “So, you’re here for me?”
+
+Leanne had faced many larger and taller than even Delphine, but something about her made Leanne’s chest tighten and her breath quiver.
+
+“So to speak,” said Leanne.
+
+“Well, introduce yourself,” said Delphine. 
+
+“Leanne.” What else could she say? She didn’t know where she was from. She couldn’t say what she did. ‘Assassin?’ ‘Pilot?’ ‘Bounty hunter,’ perhaps—it would at least be legal, and moderately truthful.
+
+“And you know me,” said Delphine.
```

Comment:
I could possibly add a bit more here, actually. "And you know me" could be followed with:

> I know you're from Dressara.
>
> My parents were from Dressara.
>
> And you?
>
> Not Dressara.

"Not Dressara" is okay, but something a little more coy might be nice.

Anyway, then:

> I know what your parents named you.